.gitignore

```diff
@@ -2,6 +2,7 @@
 coverage
 .nyc_output
 .yarn-cache
+.yarn-offline-mirror
```

Comment:
This should not be ignored - I realize that having the tgzs in source control seems odd on the surface, but I like that it allows running `yarn --offline` without requiring any further downloads.

Reply:
Ah right, I see. I've not come across this before. I just saw a tgz getting staged and freaked out and added the gitignore without looking to see these were already being committed 😄